src/library/Choice/styled.js

```diff
@@ -136,7 +136,7 @@ export const Control = styled('span', {
     borderColor: theme.ChoiceControl_borderColor,
     borderRadius: theme.ChoiceControl_borderRadius,
     borderStyle: 'solid',
-    borderWidth: '1px',
+    borderWidth: '2px',
```

Comment:
Instead of hard-coding values, I think we should create a theme setting. e.g., theme.ChoiceControl_borderWidth